src/packages/documents/document-types/workspace/views/design/document-type-workspace-view-edit.element.ts

```diff
@@ -109,34 +124,70 @@ export class UmbDocumentTypeWorkspaceViewEditElement
 		this._routes = routes;
 	}
 
-	#requestRemoveTab(tabId: string | undefined) {
+	#requestRemoveTab(tab: PropertyTypeContainerModelBaseModel | undefined) {
+		const Message: UmbConfirmModalData = {
+			headline: 'Delete tab',
+			content: html`Are you sure you want to delete the tab (${tab?.name || tab?.id})?
+				<div style="color:var(--uui-color-danger-emphasis)">
+					This will delete all items that doesn't belong to a composition.
+				</div>`,
```

Comment:
```suggestion
			content: html`<umb-localize key="contentTypeEditor_confirmDeleteTabMessage" .args=${[tab?.name ?? tab?.id]}>Are you sure you want to delete the tab <strong>${tab?.name ?? tab?.id}</strong></umb-localize>
				<div style="color:var(--uui-color-danger-emphasis)">
					<umb-localize key="contentTypeEditor_confirmDeleteTabNotice">This will delete all items that doesn't belong to a composition.</umb-localize>
				</div>`,
```